pkg/kubelet/prober/prober_manager_test.go

```diff
@@ -119,7 +118,7 @@ func TestAddRemovePods(t *testing.T) {
 
 	// Removing probed pod.
 	m.RemovePod(&probePod)
-	if err := waitForWorkerExit(m, probePaths); err != nil {
+	if err := waitForWorkerExit(t, m, probePaths); err != nil {
```

Comment:
This change for trans klog to tesing log, may be in another pr.

Reply:
I removed changes to test files from this PR. I'll create a separate PR for that.